modules/test/playwright/tests/frontend-data-set-web/main/tests/advanced/actions.spec.ts

```diff
@@ -249,4 +249,32 @@ test('Behavior of item actions', async ({fdsSamplePage, page}) => {
 
 		await waitForAlert(page);
 	});
+
+	await test.step('Check that Sample Delete action has custom className applied', async () => {
+		const tableItemActionButton =
+			fdsSamplePage.table.itemActionButtons.first();
+
+		await expect(tableItemActionButton).toBeVisible();
+
+		const dropdownId =
+			await tableItemActionButton.getAttribute('aria-controls');
+
+		await tableItemActionButton.click();
+
+		await page
+			.locator(`#${dropdownId}`)
+			.filter({has: page.getByRole('menu')})
+			.waitFor();
```

Comment:
Can we abstract this out into a function that can be called from this step as well as from the `'Check that the Item Actions dropdown is present in table row` test step?

I also see a very similar code in the `FDSSamplePage.checkDropdownMenuIconsAreVisible()` function. `DSSamplePage` looks a great place to put that function.

It could look like:
```
async getDropdownId(itemActionButton: Locator) {
		await itemActionButton.click();

		const dropdownId = await itemActionButton.getAttribute('aria-controls');

		return this.page.locator(`#${dropdownId}`);
}
``` 

And so we'd reuse that piece in several places.

Makes sense?